### DISCUSSION TITLE
When using `latex-auto-mk: false` files under `format-resources`don't get copied to execution directory

### DISCUSSION THREAD

**Discussion by kernie** [quarto-dev/quarto-cli#4888] [Q&A]
Hi!

I have defined a section
```
format-resources:
        - resources/fonts/Eurostile.ttf
        - resources/formats/pdf/eforms.sty
```
in the `pdf`area and with `latex-auto-mk: true` these get copied into the execution dir.

However, I want to establish an own compilation chain and use external latexmk... When I set this to `latex-auto-mk: false` it starts rendering but complains:
```
Error producing PDF.
! LaTeX Error: File `eforms.sty' not found.
```

Can somebody enlighten me, what I'm doing wrong? How can I get more debug output?

Thank you!

**Comment by dragonstyle** [comment]
I wasn't able to reproduce this with a simple test case (I am seeing the file being copied to the execution directory whether or not `latex-auto-mk` is true or false). Do you have a simple reproducible example project or repo you could share so I could take a closer look?

**Comment by ghost** [comment]
Moved to https://github.com/quarto-dev/quarto-cli/issues/5078
